tools/fast/series/init_and_start.go

```diff
@@ -7,11 +7,17 @@ import (
 )
 
 // InitAndStart is a quick way to run Init and Start for a filecoin process.
-func InitAndStart(ctx context.Context, node *fast.Filecoin) error {
+func InitAndStart(ctx context.Context, node *fast.Filecoin, fns ...func(context.Context, *fast.Filecoin) error) error {
```

Comment:
Maybe just a comment about at what point `fns` are intended to be run (even though it's right there in the code)